pkg/ebpf/tracer.go

```diff
@@ -302,6 +307,11 @@ func (t *Tracer) initPerfPolling() (*bpflib.PerfMap, error) {
 					return
 				}
 				atomic.AddInt64(&t.perfLost, int64(lostCount))
+			case <-flushIdle.C:
+				idleConns := t.batchManager.GetIdleConns(time.Now())
```

Comment:
You can use the result of receiving on `flushIdle.C`  instead of calling time.Now() twice

Reply:
good call 👍